css/media.css

```
@@ -196,220 +161,227 @@
+        display: none;
+    }
+    .menu__burger:focus.menu__burger {
+        top: 0;
+        transform: rotate(45deg);
+    }
+    .menu__burger:focus.menu__burger::before {
+        top: 0;
+        transform: rotate(-90deg)
+    }
+    .menu__burger:focus.menu__burger::after {
+        display: none;
+    } */
     .header__play {
         display: none;
         border: none;
     }
-
     .header__play--768 {
         display: block;
         cursor: pointer;
     }
-
     /* ------hero------ */
     .hero__img {
         display: none;
     }
-
     /* ---podcast--- */
     .podcast__list {
         flex-direction: column;
     }
-
     .podcast__list-item {
         width: 100%;
     }
-
     .podcast__list {
         grid-template-columns: 1fr;
     }
-
     /* ------guests------ */
     .guests__wrapper {
         gap: 50px;
         flex-direction: column;
     }
-
     .guests__inner-img-plug,
     .guests__inner-img {
         width: 540px;
         height: 220px;
     }
-
     .ac-text {
         gap: 30px 30px;
     }
-
     .guests__text {
         max-width: 398px;
     }
-
     /* ------broadcasts------ */
     .broadcasts__wrapper {
         gap: 0;
     }
-
     .broadcasts__inner {
         margin-bottom: 45px;
     }
-
     .broadcasts__wrap-cards {
         margin-bottom: 61px;
     }
-
     /* ------adv------ */
     .adv {
         padding-top: 54px;
     }
-
     .adv__wrapper {
         flex-direction: column;
         gap: 49px;
     }
-
     .adv__wrap-img {
         padding: 0;
     }
-
     .adv__header {
         width: 490px;
     }
-
     .adv__img {
         width: auto;
         height: auto;
     }
-
     .adv__img-svg {
         top: 0;
         width: 160px;
         height: 160px;
     }
-
     /* ------about------ */
     /* ------footer------ */
     .footer {
         padding: 93px 0 7px 0;
     }
-
     .footer__inner {
         flex-direction: column;
     }
-
     .footer__social {
         display: flex;
         flex-direction: column;
         align-items: center;
     }
-
     .social-svg {
         margin-bottom: 21px;
     }
-
     .footer__social-svg {
         width: 36px;
         height: 36px;
     }
-
     .footer__inner {
         gap: 73px;
     }
-
     .footer__menu {
         gap: 28px 157px;
     }
-
     .footer__block {
         padding-top: 79px;
     }
 }
 
+
 /* -------------------------------320px------------------------------- */
 
 @media (max-width: 767px) {
-    :root {
+     :root {
         --padding-aside: 0 32px;
     }
-
     .container {
         padding: var(--padding-aside);
     }
-
     /* ------header------ */
+    .burger__wrapper {
+        padding: 74px 34px 24px 32px;
+    }
+    .burger__list {
+        gap: 39px 32px;
+    }
+    .nav-menu {
+        flex-direction: column;
+    }
+    .burger-vector-wrapper {
+        display: none;
+    }
+    .header__list-modal {
+        display: grid;
+        padding: 32px 32px 15px 32px;
```

```suggestion
        padding: 32px 32px 15px;
```
Если слева и справа одинаковое значение (а чаще всего это должно быть именно так) то можно использовать более короткую запись